### ISSUE TITLE
Add serial tasks support

### ISSUE THREAD

**Issue by lyz-code** [lyz-code/pydo#42]
Create recurrent tasks without a date. Imagine you have a big task like summarizing a book.

This kind of task is so big that it can't be done within the scope of a sprint, so it has to be splitted in smaller tasks, such as summarize chapter 1, or summarize 30 pages.

The idea is to create a parent task with the defined attributes, and once a children task is done it will spawn the next one with the updated incremented description. So second task will be `Summarize chapter 2` or `Summarize pages 30-60`.

**Comment by xerus2000** [comment]
Yes, I always wanted this as well :) but shouldn't this be handled properly with dependencies?

**Comment by lyz-code** [comment]
Dependencies help you define relationship between tasks to block or release them given the state of other tasks.

Serial tasks, as I see them, will solve another problem. The division of a big task in equal consecutive subtasks. So they can be thought as recurrent tasks without dates associated to them. In a way that when you complete one, the next one (with the updated attributes) is automatically created. This gives an easy way to manage and follow the progress of these structured tasks. 

Throughout these years I've wasted time continuously rescheduling the due date of tasks, so I try not to define it, keeping my `due.before:tomorrow` as clean as possible. It also gives me internal peace, as I don't have a huge list of tasks that *have to* be done today. The priorization of the rest of the tasks is done through sprint or week planning and reviews, which is more flexible and friendly, as it's you who decide what to do.
